app/poi_api/service.py

```diff
@@ -0,0 +1,65 @@
+from flask import jsonify
+from typing import List, Dict
+from .poi_suggestion import POI_suggestion
+from .poi import POI
+from .errors import POINotFoundError, InvalidPOISuggestionError
+from app import common
+from datetime import datetime
+from app.firebase import firestore_db, POI_COLLECTION, POI_PROPOSAL_COLLECTION
+
+poi_collection = firestore_db.collection(POI_COLLECTION)
+poi_proposal_collection = firestore_db.collection(POI_PROPOSAL_COLLECTION)
+
+
+def list_POI() -> List[POI]:
+    return [POI.from_dict(doc.to_dict()) for doc in poi_collection.stream()]

```

Comment:
```suggestion
def list_POI() -> Iterator[POI]:
    return map(lambda doc: POI.from_dict(doc.to_dict()), poi_collection.stream()
```
It's fine (and much better on memory) for the service method to return an iterator if possible here. We don't need to convert to a list yet I think. Leave that for the api level.